src/test/store/receive-profile.test.js

```diff
@@ -209,6 +210,27 @@ describe('actions/receive-profile', function() {
       ]);
     });
 
+    it('will show screenshot if the profile has screenshot data', function() {
+      const store = blankStore();
+      let profile = _getSimpleProfile();
```

Comment:
We are loading a single profile here for this test (that also includes the part that is with `getScreenshotTrackProfile`). But this patch is about the comparison view. We are not testing the comparison view at this test.